.github/workflows/gke.yaml

```diff
@@ -85,36 +117,69 @@ jobs:
         run: |
           cilium hubble enable
 
-      - name: Status
+      - name: Wait for Cilium status to be ready
         run: |
           cilium status --wait
 
-      - name: Relay Port Forward
+      - name: Port forward Relay
         run: |
           kubectl port-forward -n kube-system deployment/hubble-relay 4245:4245&
           sleep 5s
 
-      - name: Connectivity Test
+      - name: Run connectivity test
         run: |
           cilium connectivity test
 
-      - name: Uninstall cilium
-        run: |
-          cilium uninstall --wait
```

Comment:
Is it important to also test uninstall in this job?

Reply:
This was a leftover from a previous version where we uninstall before reinstalling with encryption enabled. It shouldn't have been there in the first place since encryption is out of scope for now ;)